mono/mini/interp/interp.c

```diff
@@ -2398,7 +2401,18 @@ copy_varargs_vtstack (MonoMethodSignature *csig, stackval *sp, guchar *vt_sp)
 		vt_sp += arg_size;
 	}
 
-	return (guchar*)ALIGN_PTR_TO (vt_sp, MINT_VT_ALIGNMENT);
+	vt_sp += sizeof (gpointer);
+	vt_sp = (guchar*)ALIGN_PTR_TO (vt_sp, MINT_VT_ALIGNMENT);
+
+	((guchar**)vt_sp) [-1] = start;
```

Comment:
Note due to the backwards access, this isn't properly aligned. I'll fix I guess.

Reply:
Eh, I think it is ok.